ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/CommandDeduplicationIT.scala

```diff
@@ -19,15 +19,17 @@ import scala.concurrent.Future
 import scala.concurrent.duration.DurationInt
 import scala.util.{Failure, Success}
 
-final class CommandDeduplicationIT extends LedgerTestSuite {
+class CommandDeduplicationIT(ledgerTimeIntervalMs: Int) extends LedgerTestSuite {
```

Comment:
Please use `Duration`.

Reply:
Still unadressed.

Reply:
addressed in latest push now https://github.com/digital-asset/daml/pull/7294/commits/12064ba6e1ff7ca961da400072bf396bbf1b17da